src/main/java/frc/robot/util/TalonFXAdapter.java

```diff
@@ -149,4 +148,15 @@ public double getStatorCurrent() {
   public double getTorqueCurrent() {
     return talonFX.getTorqueCurrent().refresh().getValueAsDouble();
   }
+
+  private static void applyConfig(TalonFX talonFX, MotorOutputConfigs motorOutputConfigs) {
+    for (int i = 0; i < 5; i++) {
+      if (talonFX.getConfigurator().apply(motorOutputConfigs).isOK()) {
+        return;
+      }
+    }
+    System.out.println(
+        String.format(
+            "ERROR: Failed to apply motor output configs of TalonFX ID %d", talonFX.getDeviceID()));
```

Comment:
It would be good to use [Persistent Alerts](https://docs.wpilib.org/en/stable/docs/software/telemetry/persistent-alerts.html) to notify the operator of errors here.